core/src/main/drtn/game/GameEngine.java

```diff
@@ -291,14 +297,9 @@ private void nextPlayer() {
         if (currentPlayerID >= players.length) {
             currentPlayerID = 0;
 
-            if (phase == 4) {
```

Comment:
Fractal had effects triggering in the change player function which meant that player 1 was never effected